statusboard/src/components/Select/Select.js

```diff
@@ -11,19 +11,31 @@ export default function Select({
   selected,
   emptyOptionText,
   className,
+  inline = false,
 }) {
   return (
-    <div className={cx('form-control-container', className)}>
-      <label htmlFor={id} className="form-label form-label--inline">
+    <div
+      className={cx(
+        'form-control-container',
+        { 'form-control-container--inline': inline },
+        className
+      )}
+    >
+      <label
+        htmlFor={id}
+        className={cx('form-label', { 'form-label--inline': inline })}
+      >
         <div>{label}</div>
       </label>
       <select
         id={id}
         onChange={onChange}
-        className="form-control"
+        className={cx('form-control', { 'form-control--inline': inline })}
         value={selected}
       >
-        <option value={emptyOptionText}>{emptyOptionText}</option>
+        {emptyOptionText && (
+          <option value={emptyOptionText}>{emptyOptionText}</option>
+        )}
```

Comment:
One small thing I noticed is that the first topic is selected by default and there's no way to deselect it because the emptyoptiontext is empty.  Maybe we can check for if emptyOptionText is null rather than falsy here, to allow an empty string being passed in.